e2e/src/usecases/DappListDisplay.js

```diff
@@ -17,18 +17,9 @@ export default DappListDisplay = () => {
     }
   })
 
-  it('should show dapp bottom sheet when dapp is selected', async () => {
-    await sleep(2000)
```

Comment:
i wonder why this delay was there 🤔
could it be some kind of hack to ensure this test suite runs smoothly after the previous one?

Reply:
maybe - i've actually been trying to remove them since `sleep` seems like a brittle solution. most of the time we can fix the need for `sleep` by awaiting a specific UI change (but in cases like this one, the tests seem to function find without these sleeps)